geode-apis-compatible-with-redis/src/test/java/org/apache/geode/redis/internal/data/RedisStringTest.java

```diff
@@ -351,4 +354,50 @@ public void setExpirationTimestamp_stores_delta_that_is_stable() throws IOExcept
     o2.fromDelta(in);
     assertThat(o2).isEqualTo(o1);
   }
+
+  @Test
+  public void overheadConstants_shouldNotChange_withoutForethoughtAndTesting() {
+    assertThat(RedisString.PER_OBJECT_OVERHEAD).isEqualTo(8);
+    assertThat(RedisString.getPerStringOverhead()).isEqualTo(RedisString.PER_STRING_OVERHEAD);
+  }
+
+  @SuppressWarnings("unchecked")
+  @Test
+  public void should_calculateSize_closeToROSSize_ofIndividualInstances() {
+    String javaString;
+    for (int i = 0; i < 512; i += 8) {
+      javaString = makeStringOfSpecifiedSize(i);
+      RedisString string = new RedisString(new RedisKey(javaString.getBytes()));
+
+      int expected = reflectionObjectSizer.sizeof(string);
+      Long actual = Long.valueOf(string.getSizeInBytes());
```

Comment:
Intellij complains of `unnecessary boxing` for this as well as the similar line in the test below it... looks like it can just be cast to a `long` vs. using `Long.valueOf()`.  Apparently this is preferred due to space/time performance.